### PR TITLE
minor fix

### PR DESCRIPTION
**Pull request by dong0321**

### CODE REVIEW

File: opal/datatype/opal_datatype_pack.h [hunk 1]
```diff
@@ -27,6 +27,8 @@
     CONVERTOR->cbmemcpy( (DST), (SRC), (BLENGTH), (CONVERTOR) )
 #endif
 
+#define opal_datatype_pack_threshold 16 // for compile, use different value later
```
Comment (bosilca) [inline opal/datatype/opal_datatype_pack.h:30]: why is this here ?

Comment (dong0321) [reply opal/datatype/opal_datatype_pack.h:30]: Because you didn't define this in any file. It will cause compile error.

Comment (bosilca) [reply opal/datatype/opal_datatype_pack.h:30]: hmm, did I missed my git add. let me check.
